common/shlibs

```diff
@@ -3319,7 +3319,7 @@ libcodecore.so.0 libio.elementary.code-3.0_1
 libio.elementary.music-core.so.0 libio.elementary.music-5.0_1
 libpantheon-files-core.so.4 libio.elementary.files-4.1.4_1
 libpantheon-files-widgets.so.4 libio.elementary.files-4.1.4_1
-libwlroots.so.2 wlroots-0.5.0_1
+libwlroots.so.3 wlroots-0.6.0_1
```

Comment:
wlroots soname has been bumped, but there are revdeps that need to be rebuilt...

try `xbps-query -RX wlroots`

Reply:
wayfire is broken, they have not caught up with wlroots 0.6.0

Reply:
wayfire 0.2 is up to date with wlroots